currency/pair.go

```diff
@@ -149,3 +149,11 @@ func MatchPairsWithNoDelimiter(currencyPair string, pairs Pairs, pairFmt PairFor
 	}
 	return EMPTYPAIR, fmt.Errorf("currency %v not found in supplied pairs", currencyPair)
 }
+
+// GetFormatting returns the formatting style of a pair
+func (p Pair) GetFormatting() PairFormat {
+	return PairFormat{
+		Uppercase: p.Base.UpperCase && p.Quote.UpperCase,
```

Comment:
There could be a pair format difference here as well between base and quote 🤷. Just pointing it out don't need to change this.

Reply:
I'll run a test on having an error return for it tomorrow 🎉